easy_phi/app.py

```diff
@@ -8,12 +8,13 @@
 import tornado.ioloop
 import tornado.web
 from datetime import date
+from tornado.options import options, parse_config_file, define
 
-VERSION = "0.1"
-LICENSE = "GPL v3.0"
-PROJECT = "easy_phi"
-SERVER_PORT = 8888
-
+#deifne used tornado.options
+define("VERSION", type=str)
```

Comment:
these are not settings. Version bump by definition requires code change, so they should not be options